frontend/src/pages/modelServing/screens/metrics/MetricsPageTabs.tsx

```diff
@@ -8,24 +8,35 @@ import useDoesTrustyAICRExist from '~/concepts/trustyai/context/useDoesTrustyAIC
 import { SupportedArea, useIsAreaAvailable } from '~/concepts/areas';
 import { InferenceServiceKind } from '~/k8sTypes';
 import { TrustyInstallState } from '~/concepts/trustyai/types';
+import './MetricsPageTabs.scss';
+import useServingPlatformStatuses from '~/pages/modelServing/useServingPlatformStatuses';
+import { byName, ProjectsContext } from '~/concepts/projects/ProjectsContext';
+import { isProjectNIMSupported } from '~/pages/modelServing/screens/projects/nimUtils';
+import useMetricsPageEnabledTabs from './useMetricsPageEnabledTabs';
+import BiasConfigurationAlertPopover from './bias/BiasConfigurationPage/BiasConfigurationAlertPopover';
 import PerformanceTab from './performance/PerformanceTab';
 import BiasTab from './bias/BiasTab';
-import BiasConfigurationAlertPopover from './bias/BiasConfigurationPage/BiasConfigurationAlertPopover';
-import useMetricsPageEnabledTabs from './useMetricsPageEnabledTabs';
-
-import './MetricsPageTabs.scss';
+import NIMTab from './nim/NimTab';
 
 type MetricsPageTabsProps = {
   model: InferenceServiceKind;
 };
 
 const MetricsPageTabs: React.FC<MetricsPageTabsProps> = ({ model }) => {
+  const servingPlatformStatuses = useServingPlatformStatuses();
+  const isNIMAvailable = servingPlatformStatuses.kServeNIM.enabled;
+  const { projects } = React.useContext(ProjectsContext);
+  const project = projects.find(byName(model.metadata.namespace)) ?? null;
```

Comment:
```suggestion
  const project = projects.find(byName(model.metadata.namespace));
```
small nitpick but ` ?? null` is not needed